.github/workflows/update-deps.yml

```diff
@@ -27,5 +27,5 @@ jobs:
       name: Android SDK
     secrets:
       # If a custom token is used instead, a CI would be triggered on a created PR.
-      # api_token: ${{ secrets.CI_DEPLOY_KEY }}
```

Comment:
@bruno-garcia have you updated the keys for this repo as well?

Reply:
it should be there